src/find-packagelock.ts

```diff
@@ -0,0 +1,19 @@
+import path from 'path';
+import fs from 'fs';
+
+export const findPackagelock = (
+    { cwd }: { cwd: string } = {
+        cwd: process.cwd(),
+    },
+): string | undefined => {
+    let current = cwd;
+    let last = current;
+    do {
+        const search = path.join(current, 'package-lock.json');
+        if (fs.existsSync(search)) {
+            return search;
+        }
+        last = current;
+        current = path.dirname(current);
+    } while (current !== last);
+};
```

Comment:
I think we can just reuse the existing `findPackage` function and pass an options object:
```
export function findPackage({
   useLockfile = false,
} = {}) {
   ...
   const search = path.join(current,  useLockfile ? 'package-lock.json' ; 'package.json');
   ...
}
```

Reply:
Why thumbs down?